tests/testthat/test-utils.R

```diff
@@ -65,8 +65,8 @@ test_that("`wb_dims()` works/errors as expected with unnamed arguments", {
     wb_dims(1:10, 1:12, from_row = 2),
     wb_dims(rows = 1:10, cols = 1:12, from_row = 2)
   )
-  expect_warning(wb_dims("1", 2))
-  expect_warning(wb_dims(from_row = "C"))
+  expect_warning(wb_dims("1", 2), "should not be a character")
+  expect_error(wb_dims(from_row = "C"), "from_col/from_row must be positive integers if supplied.")
```

Comment:
Not sure why we should accept this. Should be save to fail.

Reply:
Agreed with the change. When I made the first amendment to `wb_dims()`, this passed and transformed it to a warning, agreed it should be an error instead